### PR TITLE
chore: add feishu notification

### PR DESCRIPTION
**Pull request by Ehco1996**
### What problem does this PR solve? 

push notification to feihu group by webhook

### What is changed and how it works?

* gihub action
* format action yaml
* need to set repo secrect KEY is `FEISHU_BOT_WEBHOOK_URL`

### Check List 

Tests 

 - No code
 
already tested in my repo https://github.com/Ehco1996/dm/actions/runs/1030304443

Code changes

- No code

Side effects

 - None

Related changes

 - Need to cherry-pick to the release branch

### CODE REVIEW

File: .github/workflows/binlog-999999.yml [hunk 1]
```diff
@@ -111,7 +111,19 @@ jobs:
...
+        if: ${{ failure() }}
+        uses: whatwewant/action-robot-feishu@v0.0.10
+        with:
+          url: ${{ secrets.FEISHU_BOT_WEBHOOK_URL }}
+          title: FIX ME PLEASE
+          text: |
+            repository: ${{ github.repository }}
+            committer: ${{ github.actor }}
+            compare: ${{ github.event.compare }}
+            job status: ${{ job.status }}
```
Comment (glorv) [inline .github/workflows/binlog-999999.yml:124]: Shall we add the detail page url here or these four fields already contains it

**Review by lance6716 — APPROVED** [review]

**Review by lichunzhu — APPROVED** [review]
LGTM

**Review by GMHDBJD — APPROVED** [review]
LGTM

**Pull request merged**